### PR TITLE
Feature/form page

### PR DESCRIPTION
**Pull request by anelbas**
Add initial form page (@media queries still to be made to the form. Modal completed)
Update navbar code.

### CODE REVIEW

**Review by AndrewBB-BBD — COMMENTED** [review]
This is so cool! Well done, Anel! :D 

My comments and questions:
- Text input fields: the text you type in is lighter than the text that is displayed when the field is empty. Could we switch that around so that what you type in is darker than the placeholder text? The colours are up to you.
- Hover effects: Could you please add hover effects to the buttons, like the "What is required of me" text? We could beef this up by changing both the cursor and the shade of the button when you hover on it. I'm not sure whether to make the button lighter or darker, or whether to change border colour, but an effect would be cool and we could use it throughout the app. It's also common button behaviour.
- When I load the page, the modal is always open. Should it be like this or am I doing something wrong?
- You said you'd do the media queries tomorrow. Perhaps we can hop on a call and chat about the modal?

File: Client-Side/adoptForm/adoptForm.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+</head>
+<body>
+    <script type="text/javascript" src="../navbar/navbar.js"></script>
+
+    <script>loadNavbar();</script>
+
+    <main class="mainBody">
+        <form action="" class="adoptionForm">
+            <header>Take the first step to take your<br>fluffy friend home!</header>
+            <a class="sliderLink" href="">What is required of me?</a><br><br>
```
Comment (AndrewBB-BBD) [inline Client-Side/adoptForm/adoptForm.html]: I think < a > is more for a hyperlink. Perhaps use a button with "text-decoration: underline" and "background: none"? 
I'm not so sure of this so please fight me if you want

File: Client-Side/adoptForm/adoptForm.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <label for="inputFirstName">FIRST NAME *</label><br>
+            <input type="text" name="inputFirstName" placeholder="Enter your first name"><br><br>
```
Comment (AndrewBB-BBD) [inline Client-Side/adoptForm/adoptForm.html]: < br > tag  shouldn't be used to space things outside of text. Rather use "margin-bottom: "

File: Client-Side/adoptForm/adoptForm.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <input type="text" name="inputPhoneNumber" placeholder="Enter your phone number"><br><br>
+            <label for="inputEmail">EMAIL *</label><br>
+            <input type="text" name="inputEmail" placeholder="Enter your email address"><br><br>
+            <label for="inputMessage">MESSAGE</label><br>
+            <textarea name="inputMessage" placeholder="Enter your message"></textarea><br><br>
+            <button class="backButton" onclick="history.back()">Back</button>
+            <button class="submitButton">Please contact me</button>
+        </form>
+
+        <aside class="modalContainer">
```
Comment (AndrewBB-BBD) [inline Client-Side/adoptForm/adoptForm.html]: Use < dialog > tag for the modal

Comment (anelbas) [reply Client-Side/adoptForm/adoptForm.html]: Made the change

File: Client-Side/adoptForm/adoptForm.html (hunk 1 above)
Comment (anelbas) [reply Client-Side/adoptForm/adoptForm.html]: Made the change

**Comment by anelbas** [comment]
Corrected the input text color to a darker gray and added hover effects (I used the same one on the yellow button as Maddie used on the home page. The color isn't the best though so it might be changed later). The modal is hidden on page load now.

We can chat about the media queries tomorrow. It will be in the afternoon though because I am out the whole morning.

Thanks or your pointers, they were super insightful & helpful. Appreciate it :)

**Review by nikitaabrey — APPROVED** [review]

**Review by AndrewBB-BBD — APPROVED** [review]

**Pull request merged**